src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3544,16 +3544,14 @@
      * Text of single-line or block comment.
      *
      * <pre>
-     * +--SINGLE_LINE_COMMENT
-     *         |
-     *         +--COMMENT_CONTENT
+     * |--SINGLE_LINE_COMMENT -&gt; SINGLE_LINE_COMMENT
+     *      |--COMMENT_CONTENT -&gt; COMMENT_CONTENT
      * </pre>
      *
      * <pre>
-     * +--BLOCK_COMMENT_BEGIN
-     *         |
-     *         +--COMMENT_CONTENT
-     *         +--BLOCK_COMMENT_END
+     * |--BLOCK_COMMENT_BEGIN -&gt; BLOCK_COMMENT_BEGIN
+     *      |--COMMENT_CONTENT -&gt; COMMENT_CONTENT
+     *      |--BLOCK_COMMENT_END -&gt; BLOCK_COMMENT_END
```

Comment:
please make java code examples to make it visual for user like `// comment` and `/* comment */`